control/ping.yaml

```diff
@@ -0,0 +1,93 @@
+openapi: 3.1.0
+
+info:
+  title: Ping models
+  description: >-
+    All ping schemas
+  version: ^0.0.0
+
+components:
+  schemas:
+    Ping:
+      type: object
+      properties:
+        choice:
```

Comment:
Is choice needed here?
A user should be allowed to request ipv4 and ipv6 pings to be sent in the same call.
The response is flexible enough to allow for ipv4 and/or ipv6.

Reply:
I have added the choice for v4 & v6 so that we can do client side validation for IPv4 & IPv6 address. If we make them in same call, we might miss on that and need to add the validation at the server. Please let me know if removing choice is more better option, I will go with it.